cmd/format-disk-cache.go

```diff
@@ -116,62 +122,41 @@ func createFormatCache(fsFormatPath string, format *formatCacheV1) error {
 // of format cache config
 func initFormatCache(drives []string) (formats []*formatCacheV1, err error) {
 	nformats := newFormatCacheV1(drives)
-	for i, drive := range drives {
-		// Disallow relative paths, figure out absolute paths.
-		cfsPath, err := filepath.Abs(drive)
```

Comment:
we need to keep filepath.Abs()

Reply:
the absolute path in this piece of code was not serving any purpose, is there any point of doing Abs() here?

Reply:
it is needed since we going to do write the `format.json` here.

Reply:
even if we write format.json, doing `filepath.Abs()` is not adding any value, i.e it will not matter if we create format.json using absolute or relative path.